frontend/src/components/elements/DeckGlJsonChart/DeckGlJsonChart.tsx

```diff
@@ -36,6 +36,11 @@ import withFullScreenWrapper from "src/hocs/withFullScreenWrapper"
 import withMapboxToken from "src/hocs/withMapboxToken"
 
 import { DeckGlJsonChart as DeckGlJsonChartProto } from "src/autogen/proto"
+
+import { withTheme } from "@emotion/react"
```

Comment:
nit: we usually try to sort imports based on third-party (on top) and internal ones. However, we aren't super consistent with it 😄

Reply:
Understood! will fix it shortly